src/lib/parseFile.ts

```diff
@@ -155,6 +158,40 @@ export const extractWikiLinks = (ast: Root, options?: ParsingOptions) => {
   return wikiLinks;
 };
 
+export interface Task {
+  description: string;
+  checked: boolean;
+}
+
+export const extractTasks = (ast: Root) => {
+  const nodes = selectAll("*", ast);
+  const tasks: Task[] = [];
+  nodes.map((node: any) => {
+    if (node.type === "listItem") {
```

Comment:
This looks like we will extract all list items as tasks which is not correct ....